### PR TITLE
[master < T563-FL] Add minimum system requirements

### PR DESCRIPTION
**Pull request by antoniofilipovic**
### Description

Minimum system requirements, lots of users are asking for them, so here it is.

### Pull request type

Please delete options that are not relevant and check the ones that are.

- [x] New documentation page

### Checklist:

- [x] I checked all content with Grammarly
- [x] I performed a self-review of my code
- [x] I made corresponding changes to the rest of the documentation
- [x] The build passes locally
- [x] My changes generate no new warnings or errors

### CODE REVIEW

**Comment by vpavicic** [comment]
We agreed to move this to individual pages for Linux packages

**Comment by antoniofilipovic** [comment]
> We agreed to move this to individual pages for Linux packages

Yes, done now

**Comment by gitbuda** [comment]
The point with minimum system requirements is that Memgraph can operate on devices with small memory sizes. I would put 512GB as a minimum system requirement for RAM to emphasize that you don't need much RAM to run Memgraph.

The point with recommended Memory size is that you should have a bit more RAM to fit a bigger graph. 8 or 16GB or even 32 seems like a good number there.

+ Memgraph works on ARM/M1 machines, that's a huge thing as well 😄

File: docs/installation/linux/debian-installation.md [hunk 1]
```diff
@@ -35,6 +35,19 @@ Memgraph packages are available for:
 You can also use [direct download](../direct-download-links.md) links to get the
 latest Memgraph packages. 
 
+
+## Minimum and recommended system requirements
+These are minimum system requirements for Linux machine when installing Memgraph from source.
+
+| Requirement | Description                                                                                                                                                                                 |
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CPU             | a modern x86-64 processor, server processor (recommended)  (Intel Xeon, AMD Opteron/Epyc)                                                                                                   |
```
Comment (gitbuda) [inline docs/installation/linux/debian-installation.md:44]: Memgraph also works on ARM, `(Intel Xeon, AMD Opteron/Epyc, ARM machines like Apple M1, Amazon Graviton)`

**Comment by vpavicic** [comment]
We've taken a completely different approach -> https://www.notion.so/memgraph/Memgraph-minimum-and-recommended-requirements-3bf31c416f2d4a3f91f7efac6b0f3781

**Pull request closed without merging**